pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -46,8 +46,9 @@ def connection_group():
 
     This command groupdefines commands to manage persistent WBEM connections
```

Comment:
Change "persistent WBEM connections" to "WBEM connection definitions", as agreed in PR #295.

Reply:
DONE